src/app/contexts/ToggleContext/index.test.jsx

```diff
@@ -1,214 +1,46 @@
 import React, { useContext } from 'react';
 import { render, act } from '@testing-library/react';
-import fetchMock from 'fetch-mock';
 import '@testing-library/jest-dom/extend-expect';
 import { queryByText } from '@testing-library/dom';
-import togglesConfig from '#lib/config/toggles';
 import { ToggleContext, ToggleContextProvider } from '.';
 
 // eslint-disable-next-line react/prop-types
 const TestComponent = ({ toggle, children }) => {
-  const toggleIsEnabled = useContext(ToggleContext).toggleState[toggle].enabled;
-
+  const { toggleState } = useContext(ToggleContext);
+  const toggleIsEnabled = toggleState[toggle] && toggleState[toggle].enabled;
   return toggleIsEnabled && <div>{children}</div>;
 };
 
-const shouldCallTogglesEndpoint = togglesUrl =>
-  expect(fetchMock.calls(togglesUrl).length).toBeTruthy();
-const shouldNotCallTogglesEndpoint = togglesUrl =>
-  expect(fetchMock.calls(togglesUrl).length).toBeFalsy();
-const shouldRenderAd = container =>
-  expect(queryByText(container, 'Dummy Ad Component')).toBeInTheDocument();
-const shouldNotRenderAd = container =>
-  expect(queryByText(container, 'Dummy Ad Component')).not.toBeInTheDocument();
+const mockRemoteToggles = {
+  service: 'mundo',
+  testToggle: {
+    enabled: true,
+  },
+};
 
-const getMockTogglesUrl = service =>
-  `https://mock-toggles-endpoint.bbc.co.uk/?application=simorgh&service=${service}&__amp_source_origin=https://www.test.bbc.com`;
+it('should render ads when remoteToggles are passed in that enable them', async () => {
+  let container;
 
-describe('ToggleContext with feature toggles', () => {
-  beforeAll(() => {
-    process.env.SIMORGH_APP_ENV = 'local';
-    process.env.SIMORGH_CONFIG_URL = 'https://mock-toggles-endpoint.bbc.co.uk';
+  await act(async () => {
+    container = await render(
+      <ToggleContextProvider remoteToggles={mockRemoteToggles}>
+        <TestComponent toggle="testToggle">Dummy Ad Component</TestComponent>
```

Comment:
Could we rename this (and other occurences) to `Dummy Component`

Reply:
I abandoned this file half way through for some reason! Updated

Reply:
Thanks!